flybywiresim/a32nx
# fix: prevent triggering key events for more inputs in efb

Fixes #[issue_no]

## Summary of Changes

Enhances the Input component with the coherent "FOCUS_INPUT_FIELD" and "UNFOCUS_INPUT_FIELD" events to prevent input fields from triggering keybinds in the TOD calculator page.
Furthermore, switches from the  `Input` to the `SimpleInput` component which also prevents keybindings from being triggered and improves the usability.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Open the TOD calculator page on the EFB and check that all input fields do work and do not trigger any keybinding.
2. Open the the Throttle Calibration page and verify the inputs for the deadband work and do not trigger any keybinding.
3. Enter the "Expert Mode" (by clicking on the blue pencil icon). Verify the detent values can be entered manually and no keybinds are triggered.
4. Remaining EFB pages should stay unchanged.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
